src/utils/stringUtils.js

```diff
@@ -0,0 +1,76 @@
+class StringUtils {
+
+    /**
+     * @summary
+     * Returns an empty string if its a null-ish value, otherwise returns
+     * the original string.
```

Comment:
The description is a good thing to have. This does not follow the same-line example indicated in [JS Doc for @summary](https://jsdoc.app/tags-summary.html)